packages/lodestar/src/metrics/validatorMonitor.ts

```diff
@@ -272,22 +298,28 @@ export function createValidatorMonitor(
     },
 
     // Register that the `indexed_attestation` was included in a *valid* `BeaconBlock`.
-    registerAttestationInBlock(indexedAttestation, block): void {
+    registerAttestationInBlock(indexedAttestation, block, rootCache): void {
       const data = indexedAttestation.data;
-      const delay = block.slot - data.slot - MIN_ATTESTATION_INCLUSION_DELAY;
+      const inclusionDistance = block.slot - data.slot;
+      const delay = inclusionDistance - MIN_ATTESTATION_INCLUSION_DELAY;
       const epoch = computeEpochAtSlot(data.slot);
-
+      const correctHead = ssz.Root.equals(rootCache.getBlockRootAtSlot(data.slot), data.beaconBlockRoot);
```

Comment:
We should only do this math if there's at least one validator in the monitor. You could do the pattern
```ts
let attestationStuff: Stuff | null = null
```
for (const index of indexes) {
  if (attestationStuff === null) {
    attestationStuff = computeStuff()
  }
  metrics.something(index, attestationStuff)
}